scripts/zkai_attention_kv_risc0_sequence_receipt_gate.py

```
@@ -0,0 +1,891 @@
+        {key: value for key, value in fields.items() if key not in {"statement_commitment"}},
+        "ptvm:zkai:attention-kv-sequence-statement:v1",
+    )
+    return fields
+
+
+def command_entry(command_id: str, command: tuple[str, ...]) -> dict[str, Any]:
+    executable = shutil.which(command[0])
+    entry = {"command_id": command_id, "command": list(command), "available": executable is not None, "returncode": None, "stdout": "", "stderr": ""}
+    if executable is not None:
+        try:
+            result = subprocess.run(list(command), check=False, text=True, capture_output=True, timeout=20, cwd=ROOT)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)